Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -149,6 +153,77 @@ local function build_pcn_segments(pcn, pcn_segment, display_name, length, includ
 	end
 end
 
+local function display_matrix_to_string(displayMatrix)
+	local result = ""
+
+	for i = 1, #displayMatrix do
```

Comment:
Somewhere there's a function that builds this matrix, and I'm wondering if from a performance standpoint it makes more sense to repurpose that function to also run some of this code to reduce the number of times we're looping over this stuff each tick.

Reply:
Refactored the whole display_matrix_to_string function.
I know build the binary representation of the digits at the same time as the pcn matrix is build, and at the end of the build_pcn_segments function I generate the full string for the display.